src/bitops.c

```diff
@@ -187,6 +190,52 @@ long long popcountScalar(void *s, long count) {
     return bits;
 }
 
+#if defined(__aarch64__)
+#include <arm_neon.h>
+
+/**
+ *  SIMD version of popcount for ARM NEON.
+ *  Processes data in 64-byte NEON batches, falls back to scalar for tail.
+ */
```

Comment:
We don't use doxygen comments. This very compact style is the most common:

```suggestion
/*  SIMD version of popcount for ARM NEON.
 *  Processes data in 64-byte NEON batches, falls back to scalar for tail. */
```